tests/sentry/api/endpoints/test_group_events_latest.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.testutils import APITestCase, SnubaTestCase
+from sentry.testutils.helpers.datetime import before_now, iso_format
+
+
+class GroupEventsLatestEndpointTest(APITestCase, SnubaTestCase):
+    def setUp(self):
+        super(GroupEventsLatestEndpointTest, self).setUp()
+
+        self.login_as(user=self.user)
+        project = self.create_project()
+
+        self.event_a = self.store_event(
+            data={
+                "event_id": "a" * 32,
+                "environment": "development",
+                "timestamp": iso_format(before_now(days=1)),
+                "fingerprint": ["group-1"],
+            },
+            project_id=project.id,
+        )
+        self.event_b = self.store_event(
+            data={
+                "event_id": "b" * 32,
+                "environment": "production",
+                "timestamp": iso_format(before_now(minutes=5)),
+                "fingerprint": ["group-1"],
+            },
+            project_id=project.id,
+        )
+        self.event_c = self.store_event(
+            data={
+                "event_id": "c" * 32,
+                "environment": "staging",
+                "timestamp": iso_format(before_now(minutes=1)),
+                "fingerprint": ["group-1"],
+            },
+            project_id=project.id,
+        )
+
+    def test_get_simple(self):
+        url = u"/api/0/issues/{}/events/latest/".format(self.event_a.group.id)
```

Comment:
Most of our other tests use the Django resolver to build these URLs. See https://github.com/getsentry/sentry/blob/master/tests/snuba/api/endpoints/test_organization_events_v2.py#L32

Reply:
@evanh For some reason, I wasn't able to use this form. The endpoints are duplicated via various prefix paths, and I think that's why they cannot be uniquely reversed.